src/content/ccip/tutorials/send-arbitrary-data-receipt-acknowledgment.mdx

```diff
@@ -0,0 +1,466 @@
+
+       | Function                  | Description                                                                                                                                             | Value (_Ethereum Sepolia_)                                                 |
+       | ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+       | allowlistDestinationChain | CCIP Chain identifier of the target blockchain. You can find each network's chain selector on the [ supported networks page ](/ccip/supported-networks) | <CopyText text="16015286601757825753" code/>, <CopyText text="true" code/> |
+       | allowlistSender           | The _message tracker_ contract address                                                                                                                  | Your deployed contract address, <CopyText text="true" code/>               |
+       | allowlistSourceChain      | CCIP Chain identifier of the source blockchain. You can find each network's chain selector on the [ supported networks page ](/ccip/supported-networks) | <CopyText text="16015286601757825753" code/>, <CopyText text="true" code/> |
+
+       <ClickToZoom
+         src="/images/ccip/tutorials/deployed-acknowledger-allowlist.webp"
+         alt="Chainlink CCIP - Acknowledger Allowlist Ethereum Sepolia and Message Tracker"
+         style="max-width: 70%;"
+       />
+
+    1. Open MetaMask and select the _Polygon Mumbai_ network.
+    1. For each function you expanded and filled in the arguments for, click the **transact** button to call the function. MetaMask prompts you to confirm the transaction. Wait for each transaction to succeed before calling the following function.
+
+1.  Finally, allow your _acknowledger_ contract as a sender in the _message tracker_ contract on _Ethereum Sepolia_:
+
+    1. On the **Deploy & Run Transactions** tab in Remix, expand the _message tracker_ contract in the **Deployed Contracts** section. Expand the `allowlistSender` function and fill in your _acknowledger_ contract address and <CopyText text="true" code/> as allowed.
+
+       {" "}
+
+       <ClickToZoom
+         src="/images/ccip/tutorials/deployed-message-tracker-allowlist-sender.webp"
+         alt="Chainlink CCIP - Message Tracker Allowlist Acknowledger"
+         style="max-width: 70%;"
+       />
+
+    1. Open MetaMask and select the _Ethereum Sepolia_ network.
+    1. Click **transact** to call the function. MetaMask prompts you to confirm the transaction.
+
+At this point, you have one _message tracker_ (sender) contract on _Ethereum Sepolia_ and one _acknowledger_ (receiver) contract on _Polygon Mumbai_. You sent `2` LINK to the _message tracker_ contract and `1` LINK to the _acknowledger_ contract to pay the CCIP fees.
```

Comment:
@khadni 2 LINK is too much given the destination chain and the gasLimit. You should be able to run the tutorial with 0.5 LINK in Sepolia

- This is the [transaction](https://sepolia.etherscan.io/tx/0x0bcd553f651c3c7c4f3c3abcaa7a2762c6b7ad8fe508da5aca3dd39dd01a33e5) on Ethereum Sepolia: the cost is 0.05 LINK. So with 0.5 LINK , you are already taking some margin.
- This is the [transaction](https://ccip.chain.link/msg/0x390d1cc8061a0e68625d99cacfe5e58f83094e44ddd7e5ae478a31e63299b9df) on Polygon Mumbai.

conclusion, it's fine if we use these fees
- Form sepolia to Mumbai: 0.5 LINK
- From Mumbai to Sepolia: 1 LINK